scripts/identify-web-features.js

```diff
@@ -24,6 +24,15 @@ const argv = yargs(process.argv)
 
 const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
 
+function getFeature(id) {
+  const feature = features[id];
+  if (!feature || feature.kind !== "feature") {
```

Comment:
Do you see any benefit from handling moved features? It could be done by returning `[id, data]` perhaps, so that callers that use the ID could use the new one.